src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const body = document.querySelector('tbody');
```

Comment:
```suggestion
const tableBody = document.querySelector('tbody');
```
`body` could be the tag `<body>`

the same with `head`